model-optimizer/mo/front/onnx/loader.py

```diff
@@ -81,6 +81,15 @@ def protobuf2nx(graph, pb):
         # add to a tensors map
         assert not name in data_nodes_map, 'Inconsistency between data_nodes_map and graph.nodes'
         data_nodes_map[name] = (name, 0)
+    output_ids = []
+    for outp in pb.graph.output:
+        name = str(outp.name)
+        if graph.has_node(name):
+            raise Error('Name {} of output node already exists in graph.', name)
```

Comment:
I am not sure that we should raise here. Instead we should show a warning and try to resolve this issue.

And also the message could be a bit more clear. Like this: `The graph contains a node with name "{}" which is equal to the model output tensor name. This means that ...`

Reply:
This should never happen, so the exception is for some very unique cases. And it is the same as for input names.